examples/argo/README.md

```diff
@@ -4,11 +4,12 @@ In this example, we will show you how to install our sample application *podtato
 
```

Comment:
Please add instructions how to install the lifecycle controller (also to the Makefile)